src/v1/controllers/BarcodePushController.js

```diff
@@ -53,7 +54,11 @@ export default FormController.extend({
   Footer: Footer,
 
   initialize: function() {
-    this.pollForEnrollment();
+    this.pollForEnrollment().catch((error)=>{
+      if (!(error instanceof AuthPollStopError)) {
```

Comment:
would the instance be other than `AuthPollStopError` when the polling is stopped due to some other means and throwing an error is expected?

Reply:
When polling is stopped the only error thrown is `AuthPollStopError`. The other error `AuthSdkError` thrown by [this](https://github.com/okta/okta-auth-js/blob/3f6c3365c419760e652a7fb2a29e2a0575b0d647/lib/tx/poll.ts#L92 ) seems to be a fatal one. So i guess we need to handle it and show it.

Reply:
@aarongranick-okta What do you think of above? should we handle the other errors?

Reply:
Isn't catching it and then calling `trigger('error'...` handling it? It should bubble up and be displayed somehow? This is something we can could confirm with a test.

Reply:
For now the errors are ignored. as discussed, we will be doing a cleanup based on the outcome of OKTA-324849